molecule/command/drivers.py

```diff
@@ -0,0 +1,63 @@
+#  Copyright (c) 2015-2018 Cisco Systems, Inc.
+#
+#  Permission is hereby granted, free of charge, to any person obtaining a copy
+#  of this software and associated documentation files (the "Software"), to
+#  deal in the Software without restriction, including without limitation the
+#  rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+#  sell copies of the Software, and to permit persons to whom the Software is
+#  furnished to do so, subject to the following conditions:
+#
+#  The above copyright notice and this permission notice shall be included in
+#  all copies or substantial portions of the Software.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+#  DEALINGS IN THE SOFTWARE.
+
+from __future__ import print_function
+
+import click
+import tabulate
+
+from molecule import logger
+from molecule import config
+
+LOG = logger.get_logger(__name__)
+
+
+@click.command()
+@click.pass_context
+@click.option(
+    '--format',
+    '-f',
+    type=click.Choice(['simple', 'plain']),
+    default='simple',
+    help='Change output format. (simple)',
+)
+def drivers(ctx, format):  # pragma: no cover
+    """ Lists drivers. """
+
+    drivers = [[x] for x in config.molecule_drivers(validate=True)]
+
+    headers = ['name']
+    if format == 'simple' or format == 'plain':
```

Comment:
Is this conditional required? I think format has to be one of these anyway

Reply:
Right, a copy/paste leftover from the implementation time, fixing it.